watcher/watcher.go

```diff
@@ -286,7 +286,9 @@ func (w *Watcher) reportRoomsStatusesRoutine() {
 		case <-podStateCountTicker.C:
 			w.PodStatesCount()
 		case <-roomStatusTicker.C:
-			_ = w.ReportRoomsStatuses()
+			if err := w.ReportRoomsStatuses(); err != nil {
```

Comment:
Do you think adding a "started reporting" log will help? Also, one after this for loop showing that it won't report anymore.

Reply:
Yeah, it could help. I added only in the forrange of roomDataSlice

Reply:
Add suggested logs!